src/worknet-ext/logger/WorkNetLogger.cs

```diff
@@ -0,0 +1,73 @@
+using System.Diagnostics;
+using Neo;
+using Neo.BlockchainToolkit.Persistence;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.Plugins;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+
+namespace WorkNetExt;
+
+public class WorkNetLogger : Plugin
```

Comment:
I get this is a sample plugin, but is there some way we could do something that doesn't completely duplicate what WorkNet's built in log plugin does? Write to a file maybe?